documentation/docs/infrastructure/overview.md

```diff
@@ -0,0 +1,120 @@
+# Infrastructure
+
+Besides SAP specific Jenkins library steps and out-of-the-box pipelines, project "Piper" offers also documentation and tooling to start the corresponding Jenkins server with all the configuration required to run project "Piper" pipelines.
+
+The core of the Jenkins infrastructure tooling is a set of [Docker images][docker-images].
+There is a main Docker image containing a preconfigured Jenkins and several tooling images used in the specific project "Piper" steps.
+
+The document and the linked resources explain the various ways of starting such a Jenkins server based on these Docker images.
+
+## Cx Server (Recommended)
+
+Cx Server is a life-cycle management tool to bootstrap a pre-configured Jenkins instance within minutes on your own (virtual) server.
+It uses the Docker images mentioned above.
+As it would be cumbersome to start the Docker image manually with all required parameters and sidecar images, this command line tool automates the bootstraping.
+
+### Setting up a Jenkins master
+
+For the following steps you will need a server are another machine which has Docker installed and configured.
+
+To get started, initialize the Cx Server by using this `docker run` command:
+
+```sh
+docker run -it --rm -u $(id -u):$(id -g) -v "${PWD}":/cx-server/mount/ ppiper/cx-server-companion:latest init-cx-server
+```
+
+This creates a few files in your current working directory.
+The shell script `cx-server` and the configuration file `server.cfg` are of special interest.
+
+Now, you can start the Jenkins server by using the following command:
+
+```sh
+chmod +x ./cx-server
+./cx-server start
+```
+
+For more information on the Cx Server and how to customize your Jenkins, have a look at the [Operations Guide for Cx Server][devops-docker-images-cxs-guide].
+
+### Setting up Jenkins agents
```

Comment:
I'm missing a "one large jenkins master" vs "distributed setup" section here. Any reason for omitting that?

Reply:
Done